tests/torch/nas/test_all_elasticity.py

```diff
@@ -300,10 +300,12 @@ def check_output_and_weights_after_training_step(model, ref_model, actual_optimi
                 },
                 'states_of_handlers': {
                     'depth': {
-                        'active_config': [0]
```

Comment:
According to the test, `active_config` is already presented in the state.
Seems like you don't need to add it in this PR.

Reply:
Thanks! State for all dimensions is now handled in SingleElasticityHandler.